### PR TITLE
Feature/resolve leak

### PR DESCRIPTION
**Pull request by LisaQ04**

### CODE REVIEW

File: waterlog/src/Components/BtnResolve.js [hunk 1]
```diff
@@ -1,65 +1,66 @@
...
+      return <div>Loading...</div>;
     }
+    return (
+      <div>
+        <h4>{this.state.text}</h4>
+        <button
+          onClick={() => this.resolve(leaksResolves)}
+          className="BtnResolve"
+        >
+          {" "}
```
Comment (laurent-mbuyu) [inline waterlog/src/Components/BtnResolve.js]: What is {" "} for?

File: waterlog/src/Components/BtnResolve.js [hunk 2]
```diff
@@ -1,65 +1,66 @@
...
+          Resolve{" "}
```
Comment (laurent-mbuyu) [inline waterlog/src/Components/BtnResolve.js]: What is Resolve{" "} for?

Comment (LisaQ04) [reply waterlog/src/Components/BtnResolve.js]: sorry did not see the gc, I removed it.

File: waterlog/src/Components/BtnResolve.js (hunk 1 above)
Comment (LisaQ04) [reply waterlog/src/Components/BtnResolve.js]: sorry did not see the gc, I removed it.

File: waterlog/src/Components/AlertTable.js [hunk 3]
```diff
@@ -37,22 +37,29 @@ class AlertTableComponent extends Component {
...
+					</tr>
 				</thead>
 				<tbody>
-					{leaks.map((alert) => (
+					{leaks.map(alert => (
 						<tr
-							key = {alert.id}
-							className = {alert.status === 'Fault' ? 'fault' : ''}
-							onClick = {() => (window.location.href = `alert/segment/${alert.id}`)}
+							key={alert.id}
```
Comment (RashadAkoodieRabbit) [inline waterlog/src/Components/AlertTable.js]: Why have you condensed the code? It was correctly formatted before. Keep it nice and consistent with appropriate spaces.

File: waterlog/src/Components/AlertTable.js [hunk 4]
```diff
@@ -37,22 +37,29 @@ class AlertTableComponent extends Component {
...
+							className={alert.status === "Fault" ? "fault" : ""}
```
Comment (RashadAkoodieRabbit) [inline waterlog/src/Components/AlertTable.js]: Why are you using strict comparison? Just asking

Comment (LisaQ04) [reply waterlog/src/Components/AlertTable.js]: sorry man this issues has been sorted in a pending PR.

File: waterlog/src/Components/AlertTable.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/Components/AlertTable.js]: sure thanks, will consider it next time.

File: waterlog/src/Components/AlertTable.js (hunk 4 above)
Comment (RashadAkoodieRabbit) [reply waterlog/src/Components/AlertTable.js]: Ok but I am asking you why?

Comment (LisaQ04) [reply waterlog/src/Components/AlertTable.js]: I was not coding this. I only formatted the file. My team member coded and made a separate PR.

Comment (cmanamela) [reply waterlog/src/Components/AlertTable.js]: This was used on a table row to highlight which items has the status of fault

Comment (RashadAkoodieRabbit) [reply waterlog/src/Components/AlertTable.js]: So btw, regardless of whether you wrote the code or not, it is all going into the same repository. Each PR is treated as a separate piece of code going into a master branch. Hence, we won't merge a PR in until we're happy with that specific PR. Not saying that this specific thing is an issue, but there's no such thing as 2 parallel PR's that fix things in the other. Decide between your team members which PR you would like to go in first in this case and then communicate that to us. We can't approve with a promise of it being fixed later, in the real world that can often not end up happening, then we would have just merged in "bad" code.

File: waterlog/src/Components/AlertTable.js (hunk 3 above)
Comment (RashadAkoodieRabbit) [reply waterlog/src/Components/AlertTable.js]: Definitely implement it next time :)

File: waterlog/src/Components/AlertTable.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/Components/AlertTable.js]: My bad, I will take into consideration next time.

File: waterlog/src/Components/BtnResolve.js (hunk 1 above)
Comment (laurent-mbuyu) [reply waterlog/src/Components/BtnResolve.js]: Great

**Review by RashadAkoodieRabbit — APPROVED** [review]

**Pull request merged**